DOCS/man/options.rst

```diff
@@ -5554,8 +5554,10 @@ them.
     no
         Disable any dithering done by mpv.
     auto
-        Automatic selection. If output bit depth cannot be detected, 8 bits per
-        component are assumed.
+        Automatic selection.
+        On ``vo=gpu``: if output bit depth cannot be detected, 8 bpc is assumed.
+        On ``vo=gpu-next``: with ``gpu-api=d3d11``, real on-the-wire bpc is used.
+        For other ``gpu-api``, 8 bpc is used for SDR content.
```

Comment:
`video/out/opengl/context.c` has a color_depth function which would mean this is wrong.

Reply:
I'm also wondering why we did this complicated thing when we could just have changed vo_gpu_next to do what the docs say:
>  If output bit depth cannot be detected, 8 bits per component are assumed.

wouldn't that have worked?

Reply:
> video/out/opengl/context.c has a color_depth function which would mean this is wrong.

It gives full format bit depth, so equivalent to what gpu-next was already doing.

> wouldn't that have worked?

This is what we are trying to do. And complication is only because dxgi (d3d11) can apparently give us more useful information so we can use it.

Reply:
We don't want to use 8bpc for HDR content, only for SDR. should probably also update vo_gpu if anyone cares.

dithering down to the backbuffer depth is fine for opengl because the banding issue only exists on d3d11 and vulkan, so I forgot about opengl my bad. I should update the docs.

we need to do this for vulkan specifically because the driver wouldn't do any dithering for 16/10 bit backbuffers for many people, across vendors.

Reply:
> We don't want to use 8bpc for HDR content, only for SDR. should probably also update vo_gpu if anyone cares.

Then what I'm suggesting would indeed not work.